TestON/drivers/common/cli/emulator/mininetclidriver.py

```diff
@@ -1092,49 +1107,130 @@ def getSwController( self, sw ):
             main.cleanup()
             main.exit()
 
-    def assignSwController( self, **kwargs ):
+    def assignSwController( self, sw, ip, port="6633", ptcp="" ):
         """
-           count is only needed if there is more than 1 controller"""
-        args = utilities.parse_args( [ "COUNT" ], **kwargs )
-        count = args[ "COUNT" ] if args != {} else 1
-
-        argstring = "SW"
-        for j in range( count ):
-            argstring = argstring + ",IP" + \
-                str( j + 1 ) + ",PORT" + str( j + 1 )
-        args = utilities.parse_args( argstring.split( "," ), **kwargs )
+        Description:
+            Assign switches to the controllers ( for ovs use only )
```

Comment:
We have many functions in here that are only for ovs. if/when we start using other software switches, we may want to look into how we can subclass this driver and/or making these functions switch agnostic